applications/matter_bridge/README.rst

```diff
@@ -270,6 +278,32 @@ You can enable the :ref:`matter_bridge_app_bridged_support` by using the followi
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED` - For the simulated bridged device.
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT` - For the Bluetooth LE bridged device.
 
+Additionally, you can decide how many bridged devices the bridge application will support.
+The decision will make an impact on the flash and RAM memories usage, what can be verified on the compilation level.
+The application uses dynamic memory allocation and stores bridged devices objects on heap, so it may be necessary to increase the heap size using ::kconfig_option::`CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE`.
```

Comment:
```suggestion
The application uses dynamic memory allocation and stores bridged device objects on the heap, so it may be necessary to increase the heap size using the :kconfig_option:`CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE` Kconfig option.
```